drivers/disk/nvme/Kconfig

```diff
@@ -52,6 +52,13 @@ config NVME_REQUEST_TIMEOUT
 	  This sets the waiting time for a request to succeed.
 	  Do not touch this unless you know what you are doing.
 
+config NVME_PRP_PAGE_SIZE
```

Comment:
please mention this new option in release notes

Reply:
done